sample-code/python/test_ios_selectors.py assert error

## The problem
function `test_should_find_elements_by_xpath` result is 8 , not 7
 

## Environment

* Appium version : 1.14.0 
* Mac OS version : 10.14.5
* Xcode version ：11.0 beta 4
* Node.js version : v8.6.0
* Npm version : 6.10.2
* Simulator version : iOS 12.0  iPhone XR 
* Appium CLI 

## Details
```
=================================== FAILURES ===================================
_____________ TestIOSSelectors.test_should_find_elements_by_xpath ______________

self = <test_ios_selectors.TestIOSSelectors object at 0x10ab56290>
driver = <appium.webdriver.webdriver.WebDriver (session="dddb8afc-6146-4c3a-8079-952eb333586c")>

    def test_should_find_elements_by_xpath(self, driver):
        action_bar_container_elements = driver.find_elements_by_xpath('//XCUIElementTypeWindow//XCUIElementTypeButton')
>       assert 7 == len(action_bar_container_elements)
E       assert 7 == 8
E        +  where 8 = len([<appium.webdriver.webelement.WebElement (session="dddb8afc-6146-4c3a-8079-952eb333586c", element="10000000-0000-0000-...ment.WebElement (session="dddb8afc-6146-4c3a-8079-952eb333586c", element="1B000000-0000-0000-AC7F-010000000000")>, ...])

test/test_ios_selectors.py:52: AssertionError
===================== 1 failed, 4 passed in 150.54 seconds =====================

```

## Appium logs
```
[HTTP] --> POST /wd/hub/session/703ece86-c6de-4e92-8c8f-f4f5d362412f/elements
[HTTP] {"using":"xpath","value":"//XCUIElementTypeWindow//XCUIElementTypeButton"}
[debug] [W3C (703ece86)] Calling AppiumDriver.findElements() with args: ["xpath","//XCUIElementTypeWindow//XCUIElementTypeButton","703ece86-c6de-4e92-8c8f-f4f5d362412f"]
[debug] [XCUITest] Executing command 'findElements'
[debug] [BaseDriver] Valid locator strategies for this request: xpath, id, name, class name, -ios predicate string, -ios class chain, accessibility id
[debug] [BaseDriver] Waiting up to 10000 ms for condition
[debug] [WD Proxy] Matched '/elements' to command name 'findElements'
[debug] [WD Proxy] Proxying [POST /elements] to [POST http://localhost:8100/session/06ACE019-2A65-4D7B-AA65-BE6D12D3F68A/elements] with body: {"using":"xpath","value":"//XCUIElementTypeWindow//XCUIElementTypeButton"}
[debug] [WD Proxy] Got response with status 200: {"value":[{"ELEMENT":"10000000-0000-0000-C682-010000000000"},{"ELEMENT":"12000000-0000-0000-C682-010000000000"},{"ELEMENT":"13000000-0000-0000-C682-010000000000"},{"ELEMENT":"14000000-0000-0000-C682-010000000000"},{"ELEMENT":"18000000-0000-0000-C682-010000000000"},{"ELEMENT":"1B000000-0000-0000-C682-010000000000"},{"ELEMENT":"1C000000-0000-0000-C682-010000000000"},{"ELEMENT":"1E000000-0000-0000-C682-010000000000"}],"sessionId":"06ACE019-2A65-4D7B-AA65-BE6D12D3F68A","status":0}
[debug] [W3C (703ece86)] Responding to client with driver.findElements() result: [{"element-6066-11e4-a52e-4f735466cecf":"10000000-0000-0000-C682-010000000000","ELEMENT":"10000000-0000-0000-C682-010000000000"},{"element-6066-11e4-a52e-4f735466cecf":"12000000-0000-0000-C682-010000000000","ELEMENT":"12000000-0000-0000-C682-010000000000"},{"element-6066-11e4-a52e-4f735466cecf":"13000000-0000-0000-C682-010000000000","ELEMENT":"13000000-0000-0000-C682-010000000000"},{"element-6066-11e4-a52e-4f735466cecf":"14000000-0000-0000-C682-010000000000","ELEMENT":"14000000-0000-0000-C682-010000000000"},{"element-6066-11e4-a52e-4f735466cecf":"18000000-0000-0000-C682-010000000000","ELEMENT":"18000000-0000-0000-C682-010000000000"},{"element-6066-11e4-a52e-4f735466cecf":"1B000000-0000-0000-C682-010000000000","ELEMENT":"1B000000-0000-0000-C682-010000000000"},{"element-6066-11e4-a52e-4f735466cecf":"1C000000-0000-0000-C682-010000000000","ELEMENT":"1C000000-0000-0000-C682-010000000000"},{"element-6066-11e4-a52e-4f735466cecf":"1E000000-0000-0000-C682-010000000000","ELEMENT":"1E000000-0000-0000-C682-01000000000...
[HTTP] <-- POST /wd/hub/session/703ece86-c6de-4e92-8c8f-f4f5d362412f/elements 200 1328 ms - 1035

```

I could reproduce this error in my local.
And I don't think python-client causes this error from recent changes in python-client.

```
___________________________________________________________________________ TestIOSSelectors.test_should_find_elements_by_xpath ____________________________________________________________________________

self = <test_ios_selectors.TestIOSSelectors object at 0x107d4cc50>, driver = <appium.webdriver.webdriver.WebDriver (session="07b4c9ae-094b-494d-a0ac-55d554694c3e")>

    def test_should_find_elements_by_xpath(self, driver):
        action_bar_container_elements = driver.find_elements_by_xpath('//XCUIElementTypeWindow//XCUIElementTypeButton')
>       assert 7 == len(action_bar_container_elements)
E       assert 7 == 8
E        +  where 8 = len([<appium.webdriver.webelement.WebElement (session="07b4c9ae-094b-494d-a0ac-55d554694c3e", element="0D000000-0000-0000-...ment.WebElement (session="07b4c9ae-094b-494d-a0ac-55d554694c3e", element="18000000-0000-0000-4B3C-000000000000")>, ...])

test/test_ios_selectors.py:51: AssertionError
```

I'm not sure if I can just change to `8` since below commit changed from `8` to `7`.
Do you have any ideas? > @imurchie 

https://github.com/appium/appium/pull/12681/commits/210447d06aec431479bbccfb996f43b646d185d4#diff-121a7a70df2b55213ca787a8c5ca99ecL51

--- a/sample-code/python/test/test_ios_selectors.py
+++ b/sample-code/python/test/test_ios_selectors.py
@@ -48,4 +48,4 @@ class TestIOSSelectors():
 
     def test_should_find_elements_by_xpath(self, driver):
         action_bar_container_elements = driver.find_elements_by_xpath('//XCUIElementTypeWindow//XCUIElementTypeButton')
-        assert 7 == len(action_bar_container_elements)
+        assert 8 == len(action_bar_container_elements)

